src/main/resources/scripts/item command.sk

```diff
@@ -19,11 +19,11 @@ command /item <items>:
 	executable by: players
 	permission: skript.give
 	trigger:
-		player has permission "skript.give.bypassblacklist":
+		if player has permission "skript.give.bypassblacklist":
 			give arguments to player
 		else:
 			loop arguments:
-				loop-item is not blacklisted:
+				if loop-item is not blacklisted:

```

Comment:
Blacklisted is not a condition?

Reply:
Look at the top of the file, blacklisted is defined as an alias

Reply:
Oh my bad. Should probably add a comment that it’s an alias and not an inbuilt conditional then